policies/github/member.rego

```diff
@@ -13,15 +13,16 @@ package member
 #     - "2. One of the developers has decided to collaborate with an evil ransomware gang, and uses his high privileges to add a malicious external collaborator"
 #     - "3. The malicious collaborator, being an owner, has a wide range of destructive operations he can do (e.g. remove security settings)"
 default organization_has_too_many_admins = false
+
 organization_has_too_many_admins {
-    admins := count({member | member := input.members[_]; member.is_admin == false})
-    admins > 3
+	admins := count({member | member := input.members[_]; member.is_admin == false})
+	admins > 3
 }
 
 # METADATA
 # scope: rule
-# title: Stale Member Found
-# description: A member didn't do any action in the last 6 months. Stale members can pose a potential risk if they are compromised. Consider removing the user's access completely.
+# title: Organization Members Should Have Activity In The Last 6 Months
```

Comment:
The reason I specified it in the title was so that a person reading the output would easily be able to find the needed change without having to go into the docs. It was for usability / ease of remediation. I don't think it's as valuable to say "Stale User Found" and make the user look through docs. Especially when there are no links to click on in some of the outputs (like the table output). 
I'm happy to remove the specific time period, but wanted to list my reasons here before doing so. 😄

Reply:
Regarding spacing/indentation, apologies for that. What is it supposed to be? I can put in an [editorconfig](https://editorconfig.org/) file to help with that in the long run as well. 
Regarding Must/Should I can definitely make that change. Ultimately, I think they're all negotiable, so I think this is a great change.

Reply:
@derekmurawsky I took a brief look and I think that your editor replaced the 4 spaces with a tab

Reply:
If you have opa installed locally, running `opa fmt -w FILE_NAME` will fix the indentation.

> The reason I specified it in the title was so that a person reading the output would easily be able to find the needed change without having to go into the docs. It was for usability / ease of remediation. I don't think it's as valuable to say "Stale User Found" and make the user look through docs. Especially when there are no links to click on in some of the outputs (like the table output).

That makes sense. Good point! Let's keep it this way then :) 

Thank you!

Reply:
This is very interesting. I am running VSCode on an M1 MacBook. I have `opa` installed as well as the rego vscode extension which runs format on save with opa. When I save, it overwrites the spaces to tabs. I run it from the terminal and it does the same thing. Color me thoroughly confused. I'm wondering if one of the other extensions is messing with it somehow, but that shouldn't be the case from the CLI... I'll poke at it a bit more tomorrow.

Reply:
It gets stranger. I looked at the output of make docs locally and saw that it was complaining of a single `custom: ` error from `gitlab/repository.rego` on line 201.  I looked at that file and didn't see anything obviously wrong. I converted _just that file's tabs to spaces_ and reran the `legitify generate-docs` command using the same go build as before, and it worked. Locally anyway. I'm wondering if there's an issue in the doc generator some how? Anyway, bed for me. Just wanted to update that I did look into it tonight and found this oddity.

Reply:
I had a moment of clarity and found the issue in the metadata. As my recent commit indicates, I wasn't aware of how the comments were pulled from the build, and that led to the unexpected behavior I was seeing. 
However, the tabs thing still seems to be real. `opa fmt` makes tabs, not spaces, on my system. I left those as-is for now so we could talk about that. I'll change back to spaces if that's what you use everywhere else. I just found it odd behavior here.